nicoleWojewoda/quiz/q1.js

```diff
@@ -0,0 +1,47 @@
+// Problem Statement
+// Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of size ‘k’.
+
+/*
+inputs: an array of positive numbers and a positive number k
+outputs: maximum sum of a contiguous subarray of size k
+
+edge case: if k is larger than the size of the given array, return 0
+
+steps:
+init start, end, max sum, and current sum
+loop through the nums array
+    add the number at the end index to the current sum
+    if end is greater than or equal to one less than k (exceeded window size),
+        compare current and max to set a new max
+        subtract the start num from current and move the window forward
+    increment the end
+return the max
+*/
```

Comment:
very clear and concise high level overview, please add space and time for future